AirCasting/MainTabBarView.swift

```diff
@@ -138,6 +147,10 @@ class EmptyDashboardButtonTapped: ObservableObject {
     @Published var mobileWasTapped = false
 }
 
+class FinishAndSyncButtonTapped: ObservableObject {
+    @Published var finishAndSyncButtonWasTapped = false
```

Comment:
Do we need this variable? It's never being changed by anything.

Reply:
It is changed in `StandaloneSessionCardView`